solr/core/src/test/org/apache/solr/cluster/placement/impl/PlacementPluginIntegrationTest.java

```diff
@@ -124,25 +124,6 @@ public void testConfigurationInSystemProps() {
     cc.shutdown();
   }
 
-  @Test
-  public void testConfigurationInSolrXml() {
```

Comment:
Has this test be moved elsewhere? It is the only end-to-end (well, kindof) verification that the placement plugin configured in solr.xml is indeed taking into account in CoreContainer.

Reply:
Yes, see `NodeConfigPlacementPluginTest`. The test requires a ZK aware SolrCloud now because it uses the ContainerPluginRegistry.